ts/features/idpay/configuration/store/actions/index.ts

```diff
@@ -0,0 +1,37 @@
+import { ActionType, createAsyncAction } from "typesafe-actions";
+import { InstrumentListDTO } from "../../../../../../definitions/idpay/InstrumentListDTO";
+import { NetworkError } from "../../../../../utils/errors";
+
+export type IdPayInitiativeInstrumentsGetPayloadType = {
+  initiativeId: string;
+};
+
+export const idpayInitiativeInstrumentsGet = createAsyncAction(
+  "IDPAY_INITIATIVE_PAYMENT_METHODS_REQUEST",
+  "IDPAY_INITIATIVE_PAYMENT_METHODS_SUCCESS",
+  "IDPAY_INITIATIVE_PAYMENT_METHODS_FAILURE"
```

Comment:
Just for consistency:

```suggestion
  "IDPAY_INITIATIVE_INSTRUMENTS_GET_REQUEST",
  "IDPAY_INITIATIVE_INSTRUMENTS_GET_SUCCESS",
  "IDPAY_INITIATIVE_INSTRUMENTS_GET_FAILURE"
```

Reply:
Addressed into b0018aed4846f147d1f309915b3f39838e220179